pycaret/containers/models/time_series.py

```diff
@@ -1665,46 +1647,46 @@ def _set_tune_distributions(self) -> Dict[str, List[Any]]:
         return tune_distributions
 
 
-class LarsCdsDtContainer(CdsDtContainer):
-    id = "lar_cds_dt"
-    name = "Least Angular Regressor w/ Cond. Deseasonalize & Detrending"
-    active = True  # set back to True as the parent has False
-    model_type = TSModelTypes.LINEAR
-
-    def return_regressor_class(self):
-        from sklearn.linear_model import Lars
-
-        return Lars
-
-    @property
-    def _set_tune_grid(self) -> Dict[str, List[Any]]:
-        tune_grid = {
-            "sp": [self.sp],
-            "deseasonal_model": ["additive"],
-            "degree": [1],
-            "window_length": [10],
-            "regressor__eps": [0.0001, 0.001, 0.01, 0.1],
-            "regressor__fit_intercept": [True, False],
-        }
-        return tune_grid
-
-    @property
-    def _set_tune_distributions(self) -> Dict[str, List[Any]]:
-        tune_distributions = {
-            "sp": CategoricalDistribution(
-                values=[self.sp, 2 * self.sp]
-            ),  # TODO: 'None' errors out here
-            "deseasonal_model": CategoricalDistribution(
-                values=["additive", "multiplicative"]
-                if self.strictly_positive
-                else ["additive"]
-            ),
-            "degree": IntUniformDistribution(lower=1, upper=10),
-            "window_length": IntUniformDistribution(lower=self.sp, upper=2 * self.sp),
-            "regressor__eps": UniformDistribution(0.00001, 0.1, log=True),
-            "regressor__fit_intercept": CategoricalDistribution(values=[True, False]),
-        }
-        return tune_distributions
+# class LarsCdsDtContainer(CdsDtContainer):
```

Comment:
do we want this all commented out? why not removed?

Reply:
Ideally, I want to include that, but don't have time to investigate it right now. Hence commented.